test/integration-test/Cargo.toml

```diff
@@ -27,6 +27,7 @@ test-case = { workspace = true }
 test-log = { workspace = true, features = ["log"] }
 tokio = { workspace = true, features = ["macros", "rt-multi-thread", "time"] }
 xdpilone = { workspace = true }
+chrono = { workspace = true }
```

Comment:
Can't we use std::time::Instant instead of this? Not a big deal, but we should try to keep integration test build times under control

Reply:
Yep done!